tidy3d/components/monitor.py

```diff
@@ -1074,6 +1074,21 @@ def storage_size(self, num_cells: int, tmesh: ArrayFloat1D) -> int:
         return BYTES_COMPLEX * len(self.theta) * len(self.phi) * len(self.freqs) * 6
 
 
+class DirectivityMonitor(FieldProjectionAngleMonitor, FluxMonitor):
+    """:class:`Monitor` that samples electromagnetic near fields in the frequency domain,
+    projects them at given observation angles,  and computes directivity. [TODO]
```

Comment:
Update description to describe directivity here? and remove [TODO] and extra space before "and"